test/Vault.fork.spec.ts

```diff
@@ -177,13 +169,13 @@ describe('Vault (fork tests)', () => {
         }),
       );
 
-      const expectedUnderlyingAmount = '998093178708890943065';
+      const expectedUnderlyingAmount = '995643944707865674480';
```

Comment:
How come these values had to change?

Reply:
For fork block https://github.com/lindy-labs/sc_solidity-contracts/pull/146/files#diff-fee31c02bb14a8fb855da739500d98954e2d30e6f9bc18b6e8a014447a5f72b1R21 `14449700` the exchange rate between DAI and LUSD is different than the rate between DAI and LUSD because the fork test points to the LUSD 3CRV pool now

Reply:
👍